sites/jeromefitzgerald.com/next.config.js

```diff
@@ -18,7 +18,7 @@ const transpilePackages = ['@jeromefitz/shared', 'next-notion']
 process.env.DESIGN_SYSTEM__TRANSPILE === 'true' &&
   transpilePackages.push('@jeromefitz/design-system')
 
-const { setupBuildInfo } = require('./scripts/build-info')
+// const { setupBuildInfo } = require('./scripts/build-info')
```

Comment:
Yea, cannot use the `GH_TOKEN` locally for some reason. And when you run `build` as much and re-run the `next.config` as much as we did here you need to turn this off 😆